src/lib/pubkey/kyber/kyber_common/kyber_types.h

```diff
@@ -14,12 +14,21 @@
 
 #include <botan/secmem.h>
 #include <botan/strong_type.h>
+#include <botan/internal/kyber_polynomial.h>
+#include <botan/internal/pqcrystals.h>
 
 #include <variant>
 #include <vector>
 
 namespace Botan {
 
+using KyberPolyNTT = Botan::CRYSTALS::Polynomial<KyberPolyTraits, Botan::CRYSTALS::Domain::NTT>;
+using KyberPolyVecNTT = Botan::CRYSTALS::PolynomialVector<KyberPolyTraits, Botan::CRYSTALS::Domain::NTT>;
+using KyberPolyMat = Botan::CRYSTALS::PolynomialMatrix<KyberPolyTraits>;
+
+using KyberPoly = Botan::CRYSTALS::Polynomial<KyberPolyTraits, Botan::CRYSTALS::Domain::Normal>;
+using KyberPolyVec = Botan::CRYSTALS::PolynomialVector<KyberPolyTraits, Botan::CRYSTALS::Domain::Normal>;
```

Comment:
As mentioned before: it would be good if these could be the basis for strong types to disambiguate them in interfaces E.g. to replace the multiple `poly_pack_t0`, `..._t1`, ... methods in `dilithium_algos.cpp` with an overloaded `poly_pack()` that just "does the right thing" ™ based on the strong type.